resources/vbmc/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM docker.io/centos:centos8
+
+RUN dnf install -y python3 python3-requests python3-pip && \
+     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python3 - -b master current-tripleo --no-stream && \
+     dnf update -y && \
```

Comment:
can we please change this to "dnf upgrade"?
the update command is  deprecated

Reply:
absolutely

Reply:
@elfosardo done